rust-connectors/sources/http/tests/get-test-full-config.yaml

```diff
@@ -0,0 +1,13 @@
+version: v1
+connector_version: dev
```

Comment:
Change this to
```
version: dev
```

This key was recently renamed.

Reply:
:+1: thanks :)